CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/email/SendToEmailClientPlugin.java

```diff
@@ -86,18 +86,22 @@ public PluginParameters createParameters() {
     }
 
     @Override
-    protected void execute(PluginGraphs graphs, PluginInteraction interaction, PluginParameters parameters) throws InterruptedException, PluginException {
+    protected void execute(final PluginGraphs graphs, final PluginInteraction interaction, final PluginParameters parameters) throws InterruptedException, PluginException {
         final String toEmail = parameters.getStringValue(TO_EMAIL_PARAMETER_ID);
         final String ccEmail = parameters.getStringValue(CC_EMAIL_PARAMETER_ID);
         final String subject = parameters.getStringValue(SUBJECT_PARAMETER_ID);
         final String body = parameters.getStringValue(BODY_PARAMETER_ID);
 
         try {
             final StringBuilder sb = new StringBuilder();
-            sb.append("mailto:").append(toEmail)
-                    .append("?subject=").append(encodeString(subject))
-                    .append("&cc=").append(encodeString(ccEmail))
-                    .append("&body=").append(encodeString(body));
+            sb.append("mailto:");
+            sb.append(toEmail);
+            sb.append("?subject=");
+            sb.append(encodeString(subject));
```

Comment:
think the appends can be called without explicitly calling `sb` each time

Reply:
Reverting this, I find it easier to read but it could just be me